tests/e2e-openshift-ossm/ossm-monolithic-otel/verify-traces.yaml

```
@@ -15,7 +15,7 @@ spec:
         - |
             TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
             KIALI_URL=https://kiali.istio-system.svc.cluster.local:20001
-            START_MICROS=1712566368978000
+            START_MICROS=1713780349850000
```

Is this timestamp really static, or should we use something like
```suggestion
            START_MICROS=$(date +%s)000000
```
instead?

---

The test image uses Alpine so had to use this to generate the START_MIRCOS. 

`START_MICROS=$(awk "BEGIN {print int($(date +%s) - 60*60)*1000000}")`

Its working well. 

```
--- PASS: chainsaw (968.69s)
    --- PASS: chainsaw/ossm-monolithic-otel (304.68s)
    --- PASS: chainsaw/ossm-tempostack (318.92s)
    --- PASS: chainsaw/ossm-tempostack-otel (345.08s)
PASS
Tests Summary...
- Passed  tests 3
- Failed  tests 0
- Skipped tests 0
Done.
```